File: qutebrowser/userscripts/darkreader.py
#!/usr/bin/env python
Dark_reader_sites = [ "*://www.netflix.com" ]
qute_config = os.environ['QUTE_CONFIG_DIR']
_darkreaderfile = qute_config + "/greasemonkey/darkreader.js"
print(_darkreaderfile)
f = open(_darkreaderfile, "w")
def listToString(s): 
    
    # initialize an empty string
    _include_temp="\n// @include       "
    str1 = "" 
    
    # traverse in the string  
    for ele in s: 
        str1 = _include_temp + ele + str1   
    
    # return string  
    return str1
Dark_reader_sites_str = listToString(Dark_reader_sites)
Dark_reader="// ==UserScript==\n// @name          Dark Reader (Unofficial)\n// @icon          https://darkreader.org/images/darkreader-icon-256x256.png\n// @namespace     DarkReader\n// @description	 Inverts the brightness of pages to reduce eye strain\n// @version       4.7.15\n// @author        https://github.com/darkreader/darkreader#contributors\n// @homepageURL   https://darkreader.org/ | https://github.com/darkreader/darkreader\n// @run-at        document-end\n// @grant         none\n// @include       http*" + Dark_reader_sites_str + "\n// @require       https://cdn.jsdelivr.net/npm/darkreader/darkreader.min.js\n// @noframes\n// ==/UserScript==\nDarkReader.enable({\n   brightness: 100,\n  contrast: 100,\n    sepia: 0\n});"
f.write(Dark_reader)
f.close

File: qutebrowser/greasemonkey/darkreader.js
// ==UserScript==
// @name          Dark Reader (Unofficial)
// @icon          https://darkreader.org/images/darkreader-icon-256x256.png
// @namespace     DarkReader
// @description	  Inverts the brightness of pages to reduce eye strain
// @version       4.7.15
// @author        https://github.com/darkreader/darkreader#contributors
// @homepageURL   https://darkreader.org/ | https://github.com/darkreader/darkreader
// @run-at        document-end
// @grant         none
// @include       http*
// @exclude       http://www.example.com/
// @exclude https://www.youtube.com/watch?v=rJtdW50NOQ0
// @exclude https://search.ononoki.org/
// @exclude https://github.com/*
// @exclude https://www.netflix.com/*
// @exclude https://browserbench.org/JetStream/
// @exclude https://browserbench.org/Speedometer2.1/
// @exclude https://browserbench.org/MotionMark1.2/
// @exclude https://browserbench.org/
// @exclude https://discord.com/*
// @exclude https://qutebrowser.org/doc/quickstart.html
// @exclude https://www.youtube.com/*
// @exclude https://wiki.greasespot.net/Include_and_exclude_rules
// @require       https://cdn.jsdelivr.net/npm/darkreader/darkreader.min.js
// @noframes
// ==/UserScript==

DarkReader.enable({
	brightness: 100,
	contrast: 100,
	sepia: 0
});
